app.ts

```diff
@@ -0,0 +1,414 @@
+    <button class="previous-records-btn">Previous</button>
+    <button class="next-records-btn">Next</button>
+    <button class="last-page-btn">Last Page</button>  
+    <button onclick="recordSelection()" id="confirmation-btn">Get Record</button>
+  </div>
+  <div class="current-page-container">
+    <p class=current-page></p>
+  </div>
+
+  `;
+}
+
+// The functionality to move through an amount of records.
+function nextPrevious(firstId: any) {
+  let navigation = document.querySelector(".navigation-btns");
+
+  let firstPage: any = document.querySelector(".first-page-btn");
+  let lastPage: any = document.querySelector(".last-page-btn");
+  let nextBtn: any = document.querySelector(".next-records-btn");
+  let previousBtn: any = document.querySelector(".previous-records-btn");
+
+  let currentPage: any = document.querySelector(".current-page");
+  let finalId: any;
+
+  if (recordNav.contains(navigation) === true) {
+    let resizeScreen = () => {
+      let amountRecord: any;
+
+      if (finalId >= 999999) {
+        if (window.innerHeight > 500) {
+          finalId = 999999;
+          firstId = finalId - 14;
+        } else if (window.innerHeight <= 500 && window.innerHeight > 350) {
+          finalId = 999999;
+          firstId = finalId - 10;
+        } else if (window.innerHeight <= 350 && window.innerHeight > 170) {
+          finalId = 999999;
+          firstId = finalId - 2;
+        } else if (window.innerHeight <= 170) {
+          finalId = 999999;
+          firstId = finalId - 1;
+        } else {
+          //pass
+        }
+      } else {
+        if (window.innerHeight > 500) {
+          amountRecord = 14;
+          finalId = firstId + amountRecord;
+        } else if (window.innerHeight <= 500 && window.innerHeight > 350) {
+          amountRecord = 10;
+          finalId = firstId + amountRecord;
+        } else if (window.innerHeight <= 350 && window.innerHeight > 170) {
+          amountRecord = 2;
+          finalId = firstId + amountRecord;
+        } else if (window.innerHeight <= 170) {
+          amountRecord = 1;
+          finalId = firstId + amountRecord;
+        } else {
+          //pass
+        }
+      }
+
+      if (firstId === 0) {
+        previousBtn.disabled = true;
+        nextBtn.disabled = false;
+      } else if (finalId === 999999) {
+        nextBtn.disabled = true;
+        previousBtn.disabled = false;
+      } else {
+        //pass
+      }
+
+      // To change the id so that the page does not cause an error
+      if (finalId > 999999 === true) {
+        let recordAmount = finalId - firstId;
+        nextBtn.disabled = true;
+        finalId = 999999;
+        firstId = finalId - recordAmount;
+
+        currentPage.innerHTML =
+          firstId +
+          " / " +
+          finalId +
+          " " +
+          (finalId - firstId + 1) +
+          " records only.";
+
+        console.log(typeof firstId, typeof finalId);
```

Comment:
I'm guessing this was for testing your code? You can remove any `console.logs` if they are not necessary.